src/Parser/GuardianDruid/Modules/Features/IronFur.js

```diff
@@ -0,0 +1,44 @@
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+
+const debug = false;
+
+// There is an issue with the logs right now, not with warcraftlogs where Iron Fur does not show stacks
+// Trying to approximate stacks by watching casts while we still have the buff
+class IronFur extends Module {
```

Comment:
Yes it would, this class was actually trying to track stacks originally but i couldn't get it working (due to a bug in Blizzards logs as they show in the UI as stacks but in the logs its one continuous buff). I will go back to trying to get this to work.